NEWS.md

```diff
@@ -1,17 +1,26 @@
 # Td Agent Builder news
 
-## Release v4.0.1 - 2020/08/18
+## Release v4.0.X - 
```

Comment:
I'm not sure the next release is 4.0.2.
Should we use **Unreleased** or something.

Reply:
I got it. Thanks.